presto-docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -211,3 +211,8 @@ Limitations
 * Dynamic filtering is currently implemented only for :doc:`/connector/hive` and :doc:`/connector/memory` connectors.
 * Push down of dynamic filters into local table scan on worker nodes is limited to broadcast joins.
 * Min-max dynamic filter collection is not supported for DOUBLE, REAL and unorderable data types.
+* When the join keys are not of the same type, dynamic filtering is supported if the
+  probe side type is wider than the build side type. When the probe side type is narrower,
```

Comment:
I'm not sure readers would understand narrower, wider. Maybe:
```
Dynamic filtering could also work if join keys are of different types and there is an implicit coercion
from probe key type to build key type. However, not all implicit coercion are supported. For example...
```

Reply:
updated